requirements.txt

```diff
@@ -1,18 +1,18 @@
 attrs==23.1.0 ; python_version >= "3.8" and python_version < "3.12"
-certifi==2022.12.7 ; python_version >= "3.8" and python_version < "3.12"
+certifi==2023.5.7 ; python_version >= "3.8" and python_version < "3.12"
 cffi==1.15.1 ; python_version >= "3.8" and python_version < "3.12" and sys_platform == "linux"
 charset-normalizer==3.1.0 ; python_version >= "3.8" and python_version < "3.12"
 colorama==0.4.6 ; python_version >= "3.8" and python_version < "3.12" and platform_system == "Windows"
 contourpy==1.0.7 ; python_version >= "3.8" and python_version < "3.12"
 cryptography==40.0.2 ; python_version >= "3.8" and python_version < "3.12" and sys_platform == "linux"
 cycler==0.11.0 ; python_version >= "3.8" and python_version < "3.12"
 dbfread==2.0.7 ; python_version >= "3.8" and python_version < "3.12"
-dicompyler-core==0.5.5 ; python_version >= "3.8" and python_version < "3.12"
-fonttools==4.39.3 ; python_version >= "3.8" and python_version < "3.12"
-greenlet==2.0.2 ; python_version >= "3.8" and platform_machine == "aarch64" and python_version < "3.12" or python_version >= "3.8" and platform_machine == "ppc64le" and python_version < "3.12" or python_version >= "3.8" and platform_machine == "x86_64" and python_version < "3.12" or python_version >= "3.8" and platform_machine == "amd64" and python_version < "3.12" or python_version >= "3.8" and platform_machine == "AMD64" and python_version < "3.12" or python_version >= "3.8" and platform_machine == "win32" and python_version < "3.12" or python_version >= "3.8" and platform_machine == "WIN32" and python_version < "3.12"
+dicompyler-core==0.5.6 ; python_version >= "3.8" and python_version < "3.12"
```

Comment:
I'll go and look at what changed.  There were some contour related compatability issues, perhaps that's been fixed.

Reply:
Thanks @sjswerdloff :slightly_smiling_face: